dev/com.ibm.ws.logging.hpel/resources/com/ibm/ws/logging/hpel/viewer/internal/resources/BinaryLogMessages.nlsprops

```diff
@@ -236,6 +236,13 @@ BL_VIEW_USAGE_033=\
 \ \ \ \ --isoDateFormat
 BL_VIEW_USAGE_034=\
 \tSpecify ISO-8601 date and time format to use for output.
+
+# do not translate "--excludeMessage=value[,value]*"
+BL_VIEW_USAGE_035=\
+\ \ \ \ --excludeMessage=value[,value]*
+BL_VIEW_USAGE_036=\
+\tExclude records with specified message.  Value may include * or ? as a\n\
```

Comment:
Please change to:

Exclude records with **the** specified message.  **The v**alue **can** include * or ? as a